README.md

```diff
@@ -87,11 +95,82 @@ class ExampleCollectionViewCellItem: CollectionViewCellItem {
 }
 ```
 
+## Diffs
+
+CollectionViewTools support "diffs". It means that `CollectionViewManager` can calculate difference between old and new section items and animate collection view sections and cells accordingly. 
+
+To work with diffs all cell items must conform `DiffItem` protocol: 
+
+```swift
+public protocol DiffItem {
+    var diffIdentifier: String { get }
+    func isEqual(to item: DiffItem) -> Bool
+}
+```
+
+where `diffIdentifier` is used to identify each item (note that it must be unique(!!!), otherwise diff algorithm can work incorrectly) and
+`isEqual` is used to compare items with same `diffIdentifier`.
+ 
+
+#### Cell item implementation
+```swift
+class ExampleCollectionViewCellItem: CollectionViewCellItem, DiffItem {
+
+	// implementation...    
```

Comment:
Here is a `tab` symbol instead of 4 spaces and I suppose that this line should be moved to the bottom of the `class`